plugin/testdata/authorized-type.yml

```diff
@@ -40,3 +40,8 @@ steps:
       files: dist/*
     when:
       event: promote
+
+---
+kind: secret
+name: username
+data: hl3v+FODjduX0UpXBHgYzPzVTppQblg51CVgCbgDk4U=
```

Comment:
**nit:** probably better to use something that looks less like an actual secret.